types/react-transition-group/index.d.ts

```diff
@@ -0,0 +1,32 @@
+// Type definitions for react-transition-group 1.1
+// Project: https://github.com/reactjs/react-transition-group
+// Definitions by: Asana <https://asana.com>,
```

Comment:
I based this on already existing typings for legacy projects (`react-addons-transition-group` and `react-addons-css-transition-group`) and didn't want to be an asshole who removes credits. :)